tests/unit/test_ip6_address.cpp

```diff
@@ -29,39 +29,47 @@
 #include <limits.h>
 
 #include "common/encoding.hpp"
+#include "net/ip4_address.hpp"
```

Comment:
Rename `test_ip6_address.cpp` to `test_ip_address.cpp`?

Reply:
Good idea. Renamed it in the latest pushed commit.